### PR TITLE
Fix default values on import page

### LINKED ISSUE #10430: Some issues in the import page
**Describe the bug**
In the import page, we have some issues.
During the installation:
- English language
- France country
image

**Additionnal information**
PrestaShop version: develop
PHP version: 7.1

### PR DESCRIPTION
**Pull request by tomlev**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Fix empty default values on import page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #10430
| How to test?  | In BO > advanced parameters > import : default values for both separators should not be blank

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11339)

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ImportController.php [hunk 1]
```diff
@@ -86,13 +86,15 @@ public function importAction(Request $request)
         $finder = $this->get('prestashop.core.import.file_finder');
         $iniConfiguration = $this->get('prestashop.core.configuration.ini_configuration');
 
+        /* NO_PROD
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ImportController.php]: useless NO_PROD, remove this comment.

**Comment by matks** [comment]
I'm sorry I think I miss something here 😥 
If the issue to to have default data on 2 fields, why not just add them in the Form Type like https://github.com/matks/PrestaShop/commit/9d05e8f83efb28b82eab3e04e6884bb0b6269749 ?

**Comment by jolelievre** [comment]
The modification @matks suggests seems more reasonable for a dimple bug fix it it does the job
This PR introduces a lot of modifications, and I'm not fan of injecting `RequestStack` in the FormDataProvider, we are twisting the form system here as it was not designed to fetch data from session, request, ...

File: src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Import/ImportFormDataProvider.php [hunk 2]
```diff
@@ -45,23 +47,45 @@
...
+    /**
+     * @var null|RequestStack current request
+     */
+    private $request;
+
+    public function __construct(SessionInterface $session, FileFinder $importFileFinder, RequestStack $requestStack)
     {
         $this->session = $session;
         $this->importFileFinder = $importFileFinder;
+        $this->request = $requestStack->getCurrentRequest();
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Import/ImportFormDataProvider.php:59]: import "import_type" only (and allow null value).

The yaml declaration is the following: "@=service('request_stack').getCurrentRequest().get('import_type')"

File: src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml [hunk 3]
```diff
@@ -148,7 +148,7 @@ services:
             - 'LogsPage'
 
     prestashop.admin.import.form_handler:
-        class: 'PrestaShop\PrestaShop\Core\Form\FormHandler'
+        class: 'PrestaShop\PrestaShop\Core\Form\ImportPageFormHandler'
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml:151]: btw it's totally OK to write its own Form Handler!

* Simple use cases => FormHandler
* Specific use cases => MyOwnFormHandler

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by tomlev** [comment]
This PR aims to be a subset of this one: https://github.com/PrestaShop/PrestaShop/pull/10120
I know some hanges are overkill alone in this PR scope, but they will not be source to so much conflicts when merging the big one
@mickaelandrieu @matks @jolelievre

**Comment by jolelievre** [comment]
alright for me, if it's part of an already merged PR it reduces the risk of any problem
although it's overkill compared to what Mathieu proposed I guess we can merge it
This juste leaves the modif @mickaelandrieu requested, the request should never be injected (if it can be avoided)
But I guess we will have to change the develop branch as well then

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
All right I agree with @tomlev 😃 moving to QA phase now

**Comment by matks** [comment]
Also Travis wants to apply lint here 🤔 
Ping @tomlev 😉

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Fixed Travis builds by runing cs fixer

**Comment by PierreRambaud** [comment]
Thanks @tomlev @Quetzacoalt91  and @marionf for the review.

**Pull request merged**